src/main/java/io/github/thebusybiscuit/slimefun4/implementation/SlimefunItems.java

```diff
@@ -79,6 +79,7 @@ private SlimefunItems() {}
     public static final SlimefunItemStack BOUND_BACKPACK = new SlimefunItemStack("BOUND_BACKPACK", HeadTexture.ENDER_BACKPACK, "&cSoulbound Backpack", "", "&7Size: &e36", BACKPACK_ID, "", LoreBuilder.RIGHT_CLICK_TO_OPEN);
     public static final SlimefunItemStack COOLER = new SlimefunItemStack("COOLER", HeadTexture.COOLER, "&bCooler", "&fAllows you to store Juices/Smoothies", "&fand automatically consumes them when you are hungry", "&fand you have this in your Inventory", "", "&7Size: &e27", BACKPACK_ID, "", LoreBuilder.RIGHT_CLICK_TO_OPEN);
     public static final SlimefunItemStack RESTORED_BACKPACK = new SlimefunItemStack("RESTORED_BACKPACK", HeadTexture.RESTORED_BACKPACK, "&eRestored Backpack", "", "&7Retrieve your lost items", BACKPACK_ID, "", LoreBuilder.RIGHT_CLICK_TO_OPEN);
+    public static final SlimefunItemStack CONTAINMENT_BACKPACK = new SlimefunItemStack("CONTAINMENT_BACKPACK", HeadTexture.CONTAINMENT_BACKPACK, "&2Containment Backpack", "", "&7Size: &e27", BACKPACK_ID, "&7Allows for storing radioactive items", "", LoreBuilder.RIGHT_CLICK_TO_OPEN);
```

Comment:
```suggestion
    public static final SlimefunItemStack CONTAINMENT_BACKPACK = new SlimefunItemStack("CONTAINMENT_BACKPACK", HeadTexture.CONTAINMENT_BACKPACK, "&2Containment Backpack", "&7Allows you to store radioactive items", "", "&7Size: &e27", BACKPACK_ID, "", LoreBuilder.RIGHT_CLICK_TO_OPEN);
```

The description should be the first in order. Also, could be more general.